lib/format.ts

```diff
@@ -1,6 +1,6 @@
-type Formatable = Error | object | string
+type Formatable = Error | Record<string, unknown> | string
```

Comment:
I was getting an eslint error when trying to use `object`; it recommended using `Record` instead. [Typescript Docs](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype)